module/Company/view/company/company/job-list.phtml

```diff
@@ -59,6 +59,8 @@ $this->headTitle($this->translate('Jobs'));
         <?php
 
 
+        // Randomize the order of companies
+        shuffle($jobList);
```

Comment:
Doing this, as I mentioned earlier, also randomizes the order on the page which shows all jobs for a company. This is very anoying in my opinion when trying to browse through all the jobs for a specific company.